src/app/cpe/project/project.component.ts

```diff
@@ -0,0 +1,66 @@
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import {NavigationService} from "../../routing-module/navigation.service";
+import {Project} from "../../repository/repository-model";
+import {ProjectRepositoryService} from "../../repository/project-repository.service";
+import {Subscription} from "rxjs";
+import {ActivatedRoute} from "@angular/router";
+
+
+@Component({
+  selector: 'app-project',
+  templateUrl: './project.component.html'
+})
+export class ProjectComponent implements OnInit, OnDestroy {
+
+  private _project: Project | undefined = undefined;
+
+  private projectRouteSubscription: Subscription;
+
+  constructor( private projectService: ProjectRepositoryService,
+               private route: ActivatedRoute,
+               private naviService: NavigationService ) {
+    this.projectRouteSubscription = new Subscription();
+  }
+
+  ngOnInit(): void {
+    this.projectRouteSubscription = this.route.params.subscribe(parameter => {
+      if (parameter['id'] && parameter['id'] !== this.project?.id) {
+        this.loadProject(parameter['id']);
+      }
+    });
+  }
+
+  get project() {
+    return this._project;
+  }
+
+  /**
+   * Load the project's data. Navigate to the projects component if there is no data for the given project id.
+   * @param projectId: The project to which is navigated.
+   */
+  loadProject(projectId: string) {
+    this.projectService.getProjectById(projectId).then(p => {
+        this._project = p;
+    }).catch(err => this.goToProjectsOverview());
```

Comment:
I think it would be better to have 404 page instead of redirecting to the list of projects.

Reply:
Or there should be a snack bar or a dialogue. Error logging and all business logic around resources must be defined, yes. I can change it of course and create a 404 component - which will not be final as well.

Reply:
I Removed any default routing or any routing on errors. All errors are logged to the console. Further error handling or business logic must be the objective of future PR's.